src/getCoinCombination.test.js

```diff
@@ -7,4 +7,21 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
-});
+  
```

Comment:
Add another combination.

For example:
 it(`should return array with '0' when '0' is entered`, () => {
    expect(getCoinCombination(0))
      .toEqual([0, 0, 0, 0]);
  });
  
  etc